InstallSpinnaker.sh

```diff
@@ -3,6 +3,7 @@
 # This script installs Spinnaker and its dependencies.
 # See http://www.spinnaker.io/docs/creating-a-spinnaker-instance
 
+TEMPDIR=$(mktemp -d installspinnaker.XXXX)
```

Comment:
Move this to before #11 (so it is after the -e and other shell config).
On second thought, I'd like to move it around #679 which is the block where all the installs happen that use this. Then the delete can go near by around #694 and the two pair "together".